File: tp-debug/miss.f90
program miss1

  implicit none
  
  integer :: i
  integer, parameter :: n=10
  real, dimension(n) ::  A
  data A/1.,2.,3.,4.,5.,6.,7.,8.,9.,10./

  write(*,*) A

  i = 2
  call foo(A,n)

end program miss1

File: tp-debug/miss.c
#include <stdio.h>

#define n 10
// void foo_(float * ,int *, int *);

int main()
{
  int i;
  double A[n];

  for (i=0; i<n; i++) {
    A[i] = (double) i+1;
  }

  i = 2;
  foo_(A, n);

  return 0;
}
